### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by the1Riddle**
just updates

### CODE REVIEW

File: .github/workflows/main.yml
Comment (rezzcode) [inline .github/workflows/main.yml:1]: updated the time

File: README.md
Comment (rezzcode) [inline README.md:1]: update

**Review by rezzcode — APPROVED** [review]

**Comment by olusiekwin** [comment]
Bro i hope the Codes are with us on this💀😂

**Review by olusiekwin — COMMENTED** [review]
LFTM

**Comment by rezzcode** [comment]
> Bro i hope the Codes are with us on this💀😂

definitely :smile:

**Pull request merged**